app/utils/transactionHandlers/SimpleTransferHandler.tsx

```diff
@@ -55,6 +55,7 @@ export default class SimpleTransferHandler
         amount,
         recipient,
         signatureAmount,
+        memo,
```

Comment:
The memo transactions should, in my opinion, have their own separate transaction handlers.